### PR TITLE
Update App Description and Quick Start in UG

### PR DESCRIPTION
**Pull request by jordankanghm**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -17,24 +17,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
```
Comment (ruo-x) [inline docs/UserGuide.md:18]: can consider adding the link to install java 11

File: docs/UserGuide.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# iVolunteer User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+iVolunteer is your dedicated application for volunteer coordination, designed with **volunteer coordinators** in mind. It combines the efficiency of a **Command Line Interface (CLI)** with the advantages of a **Graphical User Interface (GUI)**. If you're looking to enhance your volunteer coordination tasks, iVolunteer is the tool that empowers you to **get things done efficiently**. Whether you're creating events, updating details, or managing volunteers, iVolunteer streamlines the process for **swift and effective coordination**.
```
Comment (ruo-x) [inline docs/UserGuide.md:9]: wow well written!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -17,24 +17,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+   * `list` : Lists all volunteers.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a volunteer named `John Doe` to the list.
```
Comment (ruo-x) [inline docs/UserGuide.md:34]: should this be volunteer or event? Because my take on it is that 'list' will list all volunteering events...

Comment (brein62) [reply docs/UserGuide.md:34]: agreed, main focus should be on volunteering events. can probably do `volunteer add` or `volunteer list` etc. for management of individual volunteers

**Pull request closed without merging**